cfme/tests/automate/generic_objects/buttons/test_generic_class_custom_button.py

```diff
@@ -250,9 +262,8 @@ def test_ansible_playbook_generic_object_button(appliance, generic_definition):
         view.cancel.click()
 
 
-@pytest.mark.manual
-@pytest.mark.meta(blockers=[BZ(1753237)], coverage=[1753237])
-def test_generic_object_button_edit_multiple_times():
+@pytest.mark.meta(blockers=[BZ(1753237)], automates=[1753237])
+def test_generic_object_button_edit_multiple_times(button_without_group):
```

Comment:
One question : The test doesn't look like you are editing multiple times . Request is edited and checked if it is saved or not .
Do you think test name should be `test_generic_object_button_edited_request`

Reply:
Good point @sshveta I will update it to more accurately reflect the test. Thanks!